### PR TITLE
PYCBC-1082 named scopes

### PR DESCRIPTION
**Pull request by osfameron**
Also subsumed Query part of https://issues.couchbase.com/browse/DOC-8159

### CODE REVIEW

File: modules/howtos/pages/kv-operations.adoc [hunk 1]
```diff
@@ -71,8 +73,12 @@ include::howtos:example$kv_operations.py[tag=get_cas_replace]
 Writes in Couchbase are written to a single node, and from there the Couchbase Server will take care of sending that mutation to any configured replicas.
 The optional durability parameter, which all mutating operations accept, allows the application to wait until this replication (or persistence) is successful before proceeding.
 
-In Couchbase Server releases before 6.5, Durability was set with two options -- see the xref:2.5@python-sdk::durability.adoc[6.0 Durability documentation] -- covering how many replicas the operation must be propagated to and how many persisted copies of the modified record must exist.
-Couchbase Data Platform 6.5 refines these two options, with xref:6.5@server:learn:data/durability.adoc[Durable Writes] -- although they remain essentially the same in use.
+In Couchbase Server releases before 6.5, Durability was set with two options -- see the
+xref:2.7@python-sdk::durability.adoc[6.0 Durability documentation] --
```
Comment (thejcfactor) [inline modules/howtos/pages/kv-operations.adoc]: Is the xref:2.7@python-sdk implying a 2.7 version of the SDK?  Only 2.5.x exists.

Comment (osfameron) [reply modules/howtos/pages/kv-operations.adoc]: Hah, lazy assumption on my part... (we talk about 2.7 API, I assumed this should be updated to that, my bad...)
Thanks for the spot!
Reverted in  next commit.

**Review by thejcfactor — COMMENTED** [review]
Just had one comment/question on kv-operations.adoc page.  Beyond that, LGTM!

File: modules/howtos/pages/kv-operations.adoc (hunk 1 above)
Comment (thejcfactor) [reply modules/howtos/pages/kv-operations.adoc]: 😆 All good!

File: modules/hello-world/pages/overview.adoc [hunk 2]
```diff
@@ -67,11 +67,11 @@ And take a look at the xref:howtos:working-with-collections.adoc[developer previ
 .Practical Howto Docs
 
 [.content]
-Connect to our services -- xref:howtos:kv-operations.adoc[data (KV)];
```
Comment (RichardSmedley) [inline modules/hello-world/pages/overview.adoc:70]: Can you turn off this sort of auto-blank-space-attack when doing large PRs,
it makes them very noisy.

Comment (osfameron) [reply modules/hello-world/pages/overview.adoc:70]: I think it's a default setting in VSCode, can probably turn it off...
I've resorted to "Hide whitespace changes" in github view on occasion btw.

**Pull request merged**